test/test.do

```diff
@@ -323,7 +323,7 @@ ipacheckskip ${variable6}, ///
     saving("check06_test03_out.xlsx") ///
     sheetreplace
 	
-	* Test 4
+	/* Test 4

```

Comment:
Yes, there currently isn't an input sheet or data sheet uploaded corresponding to Test 4. I wasn't sure if there had been a test there before that had gone missing in some previous update or been deliberately removed as unnecessary, so I made my test Test 5 and commented that one out.